platform/ui/package.json

```diff
@@ -52,7 +52,9 @@
     "react-dnd": "9.4.0",
     "react-dnd-html5-backend": "^9.4.0",
     "react-dnd-touch-backend": "^9.4.0",
+    "react-draggable": "^4.1.0",
     "react-i18next": "^10.11.0",
+    "react-modal": "^3.11.1",
```

Comment:
Can we nuke `"react-bootstrap-modal"`?

Reply:
Not yet I think... It's being used in DicomFileUploaderModal and DicomStorePickerModal, these can be refactored in another task and then we can nuke it.

Reply:
Created: https://github.com/OHIF/Viewers/issues/1204